CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -201,7 +202,7 @@ local function CreateTopBar()
 	end
 
 	function this:UpdateBackgroundTransparency()
-		if settingsActive and not VREnabled then
```

Comment:
If we are moving away from `VREnabled` variable here, we should delete all instances of it and get rid of the variable.